variables.tf

```diff
@@ -287,6 +371,48 @@ variable "dynamodb3_local_secondary_index_map" {
 }
 
 
+variable "dynamodb3_autoscale_write_target" {
+  type        = number
+  default     = 50
+  description = "The target value for DynamoDB write autoscaling"
+}
+
+variable "dynamodb3_autoscale_read_target" {
+  type        = number
+  default     = 50
+  description = "The target value for DynamoDB read autoscaling"
+}
+
+variable "dynamodb3_autoscale_min_read_capacity" {
+  type        = number
+  default     = 5
+  description = "DynamoDB autoscaling min read capacity"
+}
+
+variable "dynamodb3_autoscale_max_read_capacity" {
+  type        = number
+  default     = 20
+  description = "DynamoDB autoscaling max read capacity"
+}
+
+variable "dynamodb3_autoscale_min_write_capacity" {
+  type        = number
+  default     = 5
+  description = "DynamoDB autoscaling min write capacity"
+}
+
+variable "dynamodb3_autoscale_max_write_capacity" {
+  type        = number
+  default     = 20
+  description = "DynamoDB autoscaling max write capacity"
+}
+
+variable "dynamodb3_enable_autoscaler" {
+  type        = string
+  default     = "true"
```

Comment:
Isn't that supposed to be `true` (without quotes) with Terraform >= 0.12?

Reply:
Right!